custom_components/garmin_connect/sensor.py

```
@@ -300,7 +318,8 @@ async def add_body_composition(self, **kwargs):
 
         """Check for login."""
         if not await self.coordinator.async_login():
-            raise IntegrationError("Failed to login to Garmin Connect, unable to update")
+            raise IntegrationError(
+                "Failed to login to Garmin Connect, unable to update")
 
         """Record a weigh in/body composition."""
```

Pylint (3.3.7)

[warning] String statement has no effect

(W0105)